myuw/dao/gws.py

```diff
@@ -78,7 +81,9 @@ class is one of (01, 02, 03, 04, 05, 06),
 
 
 def is_student():
-    return is_undergrad_student() or is_graduate_student()
+    return (is_undergrad_student() or
+            is_graduate_student() or
+            is_pce_student())
```

Comment:
this will give PCE students the new version if they access from a mobile device.  should we leave it or change back?